src/main/java/org/mastodon/views/grapher/display/GrapherSidePanel.java

```diff
@@ -430,6 +435,11 @@ public FeatureProjectionSpec getFeatureProjectionSpec( final String projectionKe
 		return null;
 	}
 
+	public ContextChooser< Spot > getContextChooser()
+	{
+		return Cast.unchecked( this.contextChooser );
+	}
```

Comment:
All good. Since we actually have a `getContextChooser()` method, we can implement the 

```
org.mastodon.views.context.HasContextChooser< T >
```
interface.

Reply:
Well actually it is not critical, since it is not a top-level view class. What do you think?

Reply:
Indeed, it is not critical.
But, actually a good idea to it. I added a commit, which does it.